crates/burn-ndarray/src/backend.rs

```diff
@@ -21,6 +21,18 @@ impl Default for NdArrayDevice {
     }
 }
 
+fn config_name() -> String {
```

Comment:
I've moved this one to the top of the file since it didn't seem right to have features config macros inside the trait directly, so this way it seemed more readable to me, less hidden surprises ;)